packages/rpclib/rpclib.5.9.0/opam

```diff
@@ -16,9 +16,9 @@ build: [
 depends: [
   "ocaml" {>= "4.04.0"}
   "alcotest" {with-test & < "1.0.0"}
-  "jbuilder"
-  "cmdliner"
-  "rresult"
+  "jbuilder" {>= "1.0+beta7"}
```

Comment:
```suggestion
  "jbuilder" {>= "1.0+beta9"}
```

Reply:
Indeed, `Add a (package ...) field for aliases` was added in beta9